bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx128-eu-donate.html

```diff
@@ -0,0 +1,115 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}{{ ftl('whatsnew-page-title') }}{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
+
+{% block body_id %}firefox-whatsnew{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_128_eu') }}
+{% endblock %}
+
+{% block experiments %}
+  {% if switch('experiment-firefox-whatsnew-128-eu') %}
+    {{ js_bundle('firefox_whatsnew_128_eu') }}
+  {% endif %}
+{% endblock %}
+
+{% if LANG == 'en-GB' %}
+  {% set main_title = "Donate to the Mozilla Foundation, the non-profit behind Firefox" %}
+
+  {% if variant == '2' %}
+    {% set main_tagline = "Firefox is a lightning-fast browser that doesn't sell you out with shady privacy policies or back doors for advertisers. This browser is backed by the Mozilla Foundation – a global movement of people working to keep the internet open and accessible to all." %}
+    {% set secondary_tagline = "You chose Firefox as your internet browser, and now we hope you’ll take the next step to reclaim the internet. Make a donation today to the Mozilla Foundation." %}
+    {% set donate_link = "https://foundation.mozilla.org/?form=Firefox-UK-B" %}
+  {% elif variant == '3' %}
+    {% set main_tagline = "Thanks for using Firefox, which is backed by the non-profit Mozilla Foundation. Now join the organization that mobilizes people worldwide to protect our privacy online, hold big tech companies accountable, and establish trustworthy AI. Together we must build the movement to ensure the future of the internet puts people first." %}
+    {% set secondary_tagline = "If you are ready to reclaim the internet, can you make a donation to the Mozilla Foundation today?" %}
+    {% set donate_link = "https://foundation.mozilla.org/?form=Firefox-UK-C" %}
+  {% endif %}
```

Comment:
Should this conditional also include an `else` as a default if none of the conditions are true? Though I guess in this case it defaults to the other template so one of these would have to be true anyway.